packages/@uppy/transloadit/src/Client.js

```diff
@@ -126,6 +131,25 @@ export default class Client {
       .catch((err) => this.#reportError(err, { assembly, file, url, type: 'API_ERROR' }))
   }
 
+  /**
+   * Update the number of expected files in an already created assembly.
+   *
+   * @param {object} assembly
+   * @param {number} tus_num_expected_upload_files
+   */
+  updateAssembly (assembly, tus_num_expected_upload_files) {
+    const url = new URL(assembly.assembly_ssl_url)
+    url.pathname = '/update_assemblies'
+    const body = JSON.stringify({
+      assembly_updates: [{
+        ...assembly,
+        tus_num_expected_upload_files,
+      }],
+    })
+    return this.#fetchJSON(url, { method: 'post', headers: this.#headers, body })
+      .catch((err) => this.#reportError(err, { url, type: 'API_ERROR' }))
+  }
```

Comment:
Probably better to not land changes that don't work yet and keep this PR only with the relevant details

Reply:
The uppy side works though, it's not like we're landing something broken, if the user is from the future (or rather if they don't update their uppy version after transloadit.com endpoint gets its update) they could use it as is hopefully.

Reply:
"Hopefully" is usually not the best strategy. There might still be a mismatch resulting in unwanted behavior. Until we can test this works exactly as we want to, with the API change, I don't see a reason to merge and pray this into a release.

Reply:
OK, working on a POC API side.

Reply:
Yeay 🎉
<img width="1076" alt="screenshot of the Cypress test, passing" src="https://user-images.githubusercontent.com/14309773/181229282-64862355-95f0-471c-be18-f68b9196acf1.png">

Reply:
Perfect, thanks for checking this 👌